File: app/src/main/java/com/github/diegofds92/simulator/MainActivity.java
package com.github.diegofds92.simulator;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.github.diegofds92.simulator.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

    }
}

File: app/src/main/java/com/github/diegofds92/simulator/MainActivity.kt
package com.github.diegofds92.simulator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.github.diegofds92.simulator.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //setContentView(R.layout.activity_main)
        //val tvHello = findViewById<TextView>(R.id.tvHello)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}